.spelling

```diff
@@ -206,6 +206,10 @@ faucet
 OpenSea
 metadada
 GörliWei
+scaffolded
+spec
+github
```

Comment:
It should be `GitHub`, hence fix and remove here.

It also should be `specification`